nerfstudio/process_data/process_data_utils.py

```diff
@@ -58,16 +58,18 @@ class CameraModel(Enum):
 }
 
 
-def list_images(data: Path) -> List[Path]:
+def list_images(data: Path, recursive: bool = False) -> List[Path]:
     """Lists all supported images in a directory
 
     Args:
         data: Path to the directory of images.
+        recursive: Whether to search check nested folders in `data`.
     Returns:
         Paths to images contained in the directory
     """
     allowed_exts = [".jpg", ".jpeg", ".png", ".tif", ".tiff"] + ALLOWED_RAW_EXTS
-    image_paths = sorted([p for p in data.glob("[!.]*") if p.suffix.lower() in allowed_exts])
+    glob = "**/[!.]*" if recursive else "[!.]*"
```

Comment:
NIT but can you rename this to `glob_str` in case someone imports glob in the future?

Reply:
Looks like you got it, thanks!